### PR TITLE
Fix #3348: [RTL] High-fi Selection Interaction View

### LINKED ISSUE #3348: [RTL] High-fi Selection Interaction View
## Exploration Player
- Selecteion Interaction view needs to support RTL.

screenshot

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR Fixes #3348 . Fixes padding for selection interaction view to support RTL.

## Screenshot LTR and RTL
Mobile

screenshot  ......  screenshot

screenshot.......screenshot

Tablet

screenshot .....  screenshot

screenshot........screenshot

screenshot ........screenshot

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout/selection_interaction_item.xml [hunk 1]
```diff
@@ -51,6 +54,9 @@
       app:allOptionsItemInputType="@{viewModel.getSelectionItemInputType()}"
       app:data="@{viewModel.choiceItems}"
       app:entityId="@{viewModel.entityId}"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
```
Comment (rt4914) [inline app/src/main/res/layout/selection_interaction_item.xml]: This constraint should be `top_to_bottomOf` -> `TextView`

File: app/src/main/res/layout/selection_interaction_item.xml [hunk 2]
```diff
@@ -13,7 +13,7 @@
       type="org.oppia.android.app.player.state.itemviewmodel.SelectionInteractionViewModel" />
   </data>
 
-  <LinearLayout
+  <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (rt4914) [inline app/src/main/res/layout/selection_interaction_item.xml:16]: Remove `orientation`

File: app/src/main/res/layout/selection_interaction_item.xml [hunk 3]
```diff
@@ -41,6 +41,9 @@
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="14sp"
       android:textStyle="italic"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
```
Comment (rt4914) [inline app/src/main/res/layout/selection_interaction_item.xml:44]: The text should not be `centre` aligned. Instead it should be `start` aligned.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@veena14cs PTAL

File: app/src/main/res/layout/selection_interaction_item.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/layout/selection_interaction_item.xml]: Done.

File: app/src/main/res/layout/selection_interaction_item.xml (hunk 2 above)
Comment (veena14cs) [reply app/src/main/res/layout/selection_interaction_item.xml:16]: Done.

File: app/src/main/res/layout/selection_interaction_item.xml (hunk 3 above)
Comment (veena14cs) [reply app/src/main/res/layout/selection_interaction_item.xml:44]: screenshot   ..... screenshot

Done.

**Review by rt4914 — APPROVED** [review]
LGTM thanks

**Pull request merged**